docs/source-2.0/aws/aws-iam.rst

```diff
@@ -18,12 +18,17 @@ Smithy are automatically inferred. These can be disabled or augmented. For
 more information, see :ref:`deriving-condition-keys`.
 
 
+.. _aws-iam_traits-principal:
+
+----------------
+Principal Traits
```

Comment:
```suggestion
Principal traits
```

Our spec uses titles like this, [see "Constraint traits"](https://smithy.io/2.0/spec/constraint-traits.html).